service/policy/obligations/obligations.proto

```diff
@@ -14,10 +13,14 @@ import "buf/validate/validate.proto";
 
 // Definitions
 message GetObligationRequest {
-    oneof identifier {
-        string id = 1;
-        string fqn = 2;
-    }
+    option (buf.validate.message).oneof = { fields: ["id", "fqn"], required: true };
+    string id = 1 [(buf.validate.field).string.uuid = true];
```

Comment:
Should we be using the common identifiers here? https://github.com/opentdf/platform/blob/75d7590ec062f822045027d4eb0b59a48bdea465/service/common/common.proto#L8